Documentation/heisenbug.rst

```diff
@@ -57,7 +70,9 @@ On 2 ranks, use:
 
 .. code-block:: bash
 
-    export OMP_NUM_THREADS=1
+    # Note that it is possible to increase OMP_NUM_THREADS 
+    # This will speed up (the rare) portions of the code running only CPUs, e.g. the wiggle factor calculation 
+    export OMP_NUM_THREADS=1 
     export OMP_PLACES="cores"
     export OMP_PROC_BIND=spread
     mpirun -n 2 --map-by ppr:1:numa:pe=2 --report-bindings ./launch ./SeisSol_RelWithDebInfo_ssm_86_cuda_6_elastic ./parameters.par
```

Comment:
Unlike stated above, this is of order 6, not order 4 (cf. the line "To run on one GPU (here with order 4, elastic) [...]")